webapp/components/ContributionForm/ContributionForm.spec.js

```diff
@@ -235,6 +235,12 @@ describe('ContributionForm.vue', () => {
           expect(mocks.$apollo.mutate).toHaveBeenCalledWith(expect.objectContaining(expectedParams))
         })
 
+        it('deleted a teaser image', async () => {
+          expectedParams.variables.imageUpload = imageUpload
+          wrapper.find(TeaserImage).vm.$emit('addTeaserImage', imageUpload)
+          wrapper.find(TeaserImage).vm.$emit('deleteImage', null)
+        })
```

Comment:
this isn't testing anything...
there must be an expectation, what happens when `deleteImage` is emitted?

Reply:
What is `expectedParams` here?